sdk/lib/crt/wine/heap.c

```diff
@@ -779,10 +822,11 @@ int CDECL MSVCRT_memcpy_s(void *dest, MSVCRT_size_t numberOfElements, const void
         return MSVCRT_ERANGE;
     }
 
-    memcpy(dest, src, count);
```

Comment:
WTF, why do they call `memmove` inside `memcpy`?

Reply:
* inside memcpy_s

Reply:
whatever :) question still remains

Reply:
Quoting the msvcrt:string test:
https://github.com/reactos/reactos/blob/d5999c6b66c4779bc71db51df4189e2a465b0ee5/modules/rostests/winetests/msvcrt/string.c#L3927-L3928

Actually, we got that wrong a few times in ReactOS history:
- The 2005 edition: https://github.com/reactos/reactos/commit/1b17f93d3a6b8cea4e6d51baac033abdbae2826d - "MSVCRT memcpy behaves like memmove for overlapping moves"
- The 2009 edition: https://github.com/reactos/reactos/commit/d41ad9a51b5bbe86e24466795584f471c131d038 - Make memcpy a duplicate of a memmove (confirmed by testing), there is a dependency on this behaviour.
- The 2012 edition: https://github.com/reactos/reactos/commit/ff6c1ce97291b7f5830c700a108608655f5ded5f - memcpy should behave exactly like memmove for Windows compatibility.